feature-examples/python/api_key/list_api_key.py

```diff
@@ -0,0 +1,91 @@
+"""Example of using the Vectara REST API to list the API Keys."""
+
+from dataclasses import dataclass
+import logging
+from typing import Union
+
+import requests
+
+
+@dataclass
+class CorpusData:
+    """CorpusData class."""
+
+    corpus_id: int
+    corpus_name: str
+
+
+@dataclass
+class KeyData:
+    """KeyData class."""
+
+    key_id: str
+    description: str
+    key_type: str
+    enabled: bool
+    corpora: list[CorpusData]
+
+
+def list_apikeys(
+    customer_id: int,
+    jwt_token: str,
+) -> tuple[Union[list[KeyData], str], bool]:
```

Comment:
I feel that the complex return type is not idiomatic and that it would be simpler for users if this just returned a list of key data:
```
def list_api_keys(customer_id: int, jwt_token: str) -> list[KeyData]
```
Then in the code:
```
    if some_failure:
        raise Exception(message)
    ...
    return keys
```
Then users of the code would write:
```
    keys = list_api_key(customer_id, jwt_token)
```
I realize this is a departure from the existing style, but I think it would be closer to what Python users are expecting. Thoughts? This could be addressed later if you prefer staying with the existing style for now.

Reply:
I don't like the convoluted return type either. I think your suggestion is good.